src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Ball ball = new Ball(new Random().nextInt(100), new ColorSupplier().getRandomColor());
```

Comment:
The same issue here. Also, '100' is a magic number, please read common mistakes carefully. ColorSupplier is also your dependency and should be a field